core/src/main/java/io/fundrequest/core/notification/domain/Notification.java

```diff
@@ -69,11 +69,12 @@ public boolean equals(Object o) {
         }
         Notification that = (Notification) o;
         return type == that.type &&
-               Objects.equals(date, that.date);
+               Objects.equals(date, that.date) &&
+               Objects.equals(blockchainEventId, blockchainEventId);
```

Comment:
shouldn't this be Objects.equals(blockchainEventId, that.blockchainEventId); ?

Reply:
Well yes it should be, I'll change it right away!